pkgs/development/python-modules/baycomp/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, numpy
+, scipy
+, scikit-learn
+, matplotlib
+}:
+
+buildPythonPackage rec {
+  pname = "baycomp";
+  version = "1.0.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-xDRywWvXzfSITdTHPdMH5KPacJf+Scg81eiNdRQpI7A=";
+  };
+
+  propagatedBuildInputs = [ numpy scipy scikit-learn matplotlib ];
+
```

Comment:
Would you please add `unittestCheckHook` to `nativeCheckInputs` and enable checkPhase?